mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/tcp/MultiThreadedSocketServer.java

```diff
@@ -262,7 +261,6 @@ public void run() {
           }
           if (clientCommand != null) {
             if (clientCommand instanceof Integer) {
-              ;
             } else {
```

Comment:
I'm not big fan of empty if() without even a comment. Could you please refactor those lines at the same time?

Reply:
Me neither. Changed